api/scpca_portal/models/original_file.py

```diff
@@ -216,6 +217,49 @@ def download_path(self) -> Path:
         """Return the remaining part of self.s3_key that's not the download_dir."""
         return self.s3_key_path.relative_to(self.download_dir)
 
+    @property
+    def local_file_path(self):
+        return settings.INPUT_DATA_PATH / self.s3_key_path
+
+    def _exchange_multiplexed_delimeter(self, output_file: str) -> Path:
+        """
+        Return the file path with the exchanged input and output multiplexed sample delimeters.
+        """
+        return Path(
+            output_file.replace(
+                common.MULTIPLEXED_SAMPLES_INPUT_DELIMETER,
+                common.MULTIPLEXED_SAMPLES_OUTPUT_DELIMETER,
+            )
+        )
```

Comment:
Lets move this to utils as a method called path_replace(path, oldValue, newValue), that converts the path to a string, replaces the values, returns a new path.